imagebuildah/build.go

```diff
@@ -1257,6 +1257,13 @@ func (b *Executor) Build(ctx context.Context, stages imagebuilder.Stages) (strin
 		stageExecutor *Executor
 		lastErr       error
 	)
+
+	//Check if we have a one line Dockerfile, if so, layers are irrelevant
+	ignoreLayers := false
```

Comment:
Couldn't we just set b.layers or b.noCache to true?

Reply:
I think we could, but I'm always hesitant to overwrite a field that was sent in as input from the caller.

Reply:
How about 
ignoreLayers =  (!b.layers && !b.noCache) || ignoreLayers) || (len(stages) < 2 && len(stages[0].Node.Children) < 2)

Then just check below 
if ignoreLayers {\}

Reply:
I'm massaged it a bit more, moving all the logic downwards.  I could combing the setting of the variable and the if statement, but I like having them separate.  Holler if you feel strongly that they should be merged.